_rules/html-page-lang-valid-bf051a.md

```diff
@@ -34,7 +34,11 @@ For each test target, the `lang` attribute has a [valid language subtag](#valid-
 
 ## Assumptions
 
-This rule assumes that the presence of a `lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.
+- This rule assumes that the presence of a `lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.
+
+- This rule assumes that user agents and assistive technologies can programmatically determine [valid language subtags](#valid-language-subtag) even if these do not conform to the [BCP 47][] syntax.
```

Comment:
This is a reasonable assumption - BCP 47 has an algorithm for determining best matching language in 'Matching of Language Tags' at the end of BCP 47:

> In the lookup scheme, the language range is progressively truncated
   from the end until a matching language tag is located

Even if an implementor hasn't read this part of the spec, it's still likely they'll implement it because it's easy to implement and fairly obvious.